README.md

```diff
@@ -21,16 +23,21 @@ import swifter
 ```
 df = pd.DataFrame({'x': [1, 2, 3, 4], 'y': [5, 6, 7, 8]})
 
+# runs on single core
+df['x2'] = df['x'].apply(lambda x: x**2)
+# runs on multiple cores
 df['x2'] = df['x'].swifter.apply(lambda x: x**2)
-df['outCol'] = df['inCol'].swifter.apply(my_func)
-df['outCol'] = df['inCol'].swifter.apply(my_func, positional_arg, keyword_arg=keyword_argval)
 
+# use swifter apply on whole dataframe
 df['agg'] = df.swifter.apply(lambda x: x.sum() - x.min())
+
 df['outCol'] = df[['inCol1'], ['inCol2']].swifter.apply(my_func)
```

Comment:
df['outCol'] = df[['inCol1', 'inCol2']].swifter.apply(my_func)

Reply:
P.S. I know I wrote this erroneously, not you, but now that I see it I would love for it to be correct :)